frontend/ui/widget/dictquicklookup.lua

```diff
@@ -304,7 +304,7 @@ function DictQuickLookup:update()
                                             })
                                         else
                                             UIManager:show(InfoMessage:new{
-                                                text = _("Saving Wikipedia page failed."),
+                                                text = _("Saving Wikipedia page failed or cancelled."),
```

Comment:
Could you make that `canceled` to be consistent?

NB Cancelled isn't wrong. It's mostly a US/UK usage thing.